src/components/InputField/InputField.module.css

```diff
@@ -22,7 +22,7 @@
 }
 
 .error-message {
-  @apply text-error text-12 font-regular font-inter tracking-semi-wide leading-16;
+  @apply text-error absolute -bottom-xl;
```

Comment:
adjust spacing to be -24px  instead of -32px, equivalent class named **'lrg'**  instead of 'xl'
and error line-height to be 16px instead of  32px
would make it look closer to mockup, 
if you agreed to change that but mockup is out of date, ignore my review 😀
Attached the input's screenshot with my suggested changes

Reply:
![image](https://user-images.githubusercontent.com/44810632/217401456-234a6b32-80fe-4f7f-bed8-3aa51d6b3803.png)